app/views/shared/tags/_curated_links_preview.html.erb

```diff
@@ -26,6 +34,14 @@
 <h4>Uncurated items <em>(not shown to user)</em></h4>
 <ul>
 <% tag.uncurated_tagged_documents.each do |item| %>
-  <li><%= item.title %></li>
+  <li class="tagged-document">
+    <% if item.base_path %>
+      <a href=<%= Plek.new.find('content-tagger') + '/taggings/lookup' + item.base_path %>>
```

Comment:
Yeah, but this ^ just redirects to the page. Since we already have the content_id of the thing we can link to it directly right?

https://content-tagger.integration.publishing.service.gov.uk/taggings/ffe22070-123b-4390-8cc4-51f9d5b5cc74